scripts/install

```diff
@@ -371,22 +331,22 @@ ynh_use_logrotate --logfile "/var/log/matrix-$app"
 # WARNING : theses command are used in INSTALL, UPGRADE
 # For any update do it in all files
 
-cp ../sources/Coturn_config_rotate.sh $final_path/
-cp ../sources/update_synapse_for_appservice.sh $final_path/
-ynh_replace_string --match_string=__APP__ --replace_string=$app --target_file="$final_path/Coturn_config_rotate.sh"
-ynh_replace_string --match_string=__APP__ --replace_string=$app --target_file="$final_path/update_synapse_for_appservice.sh"
+ynh_add_config --template="../sources/Coturn_config_rotate.sh" --destination="$final_path/Coturn_config_rotate.sh"
+ynh_add_config --template="../sources/update_synapse_for_appservice.sh" --destination="$final_path/update_synapse_for_appservice.sh"
 
 #=================================================
 # GENERIC FINALIZATION
 #=================================================
-# SETUP SSOWAT
+# SETUP PERMISSIONS
 #=================================================
-ynh_script_progression --message="Configuring SSOwat..." --weight=1
+ynh_script_progression --message="Configuring permissions..." --weight=1
+
+ynh_permission_url --permission=main --url=$domain/_matrix/cas_server.php/login --auth_header=true
+ynh_permission_update --permission=main --label="Synapse server SSO authentication" --show_tile=false --protected=true
```

Comment:
Indeed that's a bit weird because the label of the main permission is basically the display name of the app and I would think it should just be "Synapse" ...

Reply:
Fixed by b59cbcc